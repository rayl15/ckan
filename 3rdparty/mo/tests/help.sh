#!/usr/bin/env bash

cd "${0%/*}"
../mo --help
