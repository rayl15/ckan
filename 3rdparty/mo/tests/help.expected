Mo is a mustache template rendering software written in bash.  It inserts
environment variables into templates.

Simply put, mo will change {{VARIABLE}} into the value of that
environment variable.  You can use {{#VARIABLE}}content{{/VARIABLE}} to
conditionally display content or iterate over the values of an array.

Learn more about mustache templates at https://mustache.github.io/

Simple usage:

   mo [--false] [--help] [--source=FILE] filenames...

--false       - Treat the string "false" as empty for conditionals.
--help        - This message.
--source=FILE - Load FILE into the environment before processing templates.
